mod_ci/controllers.py

```diff
@@ -903,6 +907,51 @@ def show_maintenance():
     }
 
 
+@mod_ci.route('/blocked_users')
+@login_required
+@check_access_rights([Role.admin])
+@template_renderer()
+def blocked_users():
+    return {
+        'blocked_users': BlockedUsers.query.order_by(BlockedUsers.userID)
+    }
+
+
+def username(u):
+    user_id = str(u)
+    api_url = requests.get('https://api.github.com/user/' + user_id)
+    userdata = api_url.json()
+    username = userdata['login']
+    return username
+
+
+app.jinja_env.globals.update(username=username)
+
+
+def add_user_to_blacklist():
+    form = AddUsersToBlacklist()
+    if form.validate_on_submit():
+        blocked_user = BlockedUsers(form.userID.data, form.comment.data)
+        if blocked_user == BlockedUsers.query.filter_by(userID=form.userID.data).first():
+            flash('User already blocked.')
+            return redirect(url_for('ci.blocked_users'))
+        else:
+            g.db.add(blocked_user)
+            g.db.commit()
+
+
+def remove_user_from_blacklist():
+    form = RemoveUsersFromBlacklist()
+    if form.validate_on_submit():
+        blocked_user = BlockedUsers.query.filter_by(userID=form.userID.data).first()
+        if blocked_user is None:
+            flash('No such user in Blacklist')
+            return redirect(url_for('ci.blocked_users'))
+        else:
```

Comment:
If you have a return in the if, you don't need this else; just do no else and remove the indendation.